src/main/java/com/internet/shop/model/Order.java

```diff
@@ -0,0 +1,45 @@
+package com.internet.shop.model;
+
+import java.util.List;
+
+public class Order {
+    private Long orderId;
+    private List<Product> products;
+    private Long userId;
+
+    public Order(List<Product> products, Long userId) {
+        this.products = products;
```

Comment:
we will not receive `products` from outside
remove this from method arguments and initialize as empty `ArrayList`